.github/workflows/build.yml

```diff
@@ -52,10 +52,10 @@ jobs:
         run: |
           if grep -q authToken ~/.config/cachix/cachix.dhall; then
             echo "Cachix token is present"
-            cachix watch-exec nixpkgs-terraform nix -- flake check
+            cachix watch-exec nixpkgs-terraform nix -- flake check --max-jobs 2
```

Comment:
Isn't this defaulting to the max available cores?

Reply:
@oscar-izval Yes, but I had to limit the number of concurrent builds to avoid running out of space:

<img width="1047" alt="image" src="https://github.com/user-attachments/assets/89b1e332-94f4-4689-9800-bc59fc5098c3">

Reference: 

https://github.com/stackbuilders/nixpkgs-terraform/actions/runs/10237045402/job/28319853313